### PR TITLE
LPS-130362 Migrate selection modals in commerce-product-type-virtual-web

### PR DESCRIPTION
**Pull request by jonmak08**
Hey @liferay-commerce,

Attached is an update for http://issues.liferay.com/browse/LPS-130362.

**Test Plan:**

1. Login to Liferay
2. Click on the **Control Menu**
3. Go to **Commerce** -> **Product Management** -> **Catalogs**
4. Click on the `+` to Add a Catalog
5. Click on the **Control Menu**
6. Go to **Commerce** -> **Product Management** -> **Products**
7. Click on the `+` and select **Virtual**
8. Add a **Product Name** and select the newly created Catalog
9. Click on **Virtual** tab
10. Under **Details** there is the **Select** button to open a popup selection
11. Under **Sample** check **Enable Sample**
12. Click the **Select** button to open a popup selection
13. Under **Terms of Use** enable **Enable Terms of Use**
14. Click the **Select Web Content** button to open a popup selection

commerce

Please let me know if you have any questions. Thanks!

### CODE REVIEW

File: modules/apps/commerce/commerce-product-type-virtual-web/src/main/resources/META-INF/resources/details.jspf [hunk 1]
```diff
@@ -59,76 +59,88 @@
...
+<aui:script sandbox="<%= true %>">
+	const fileEntryNameInput = document.getElementById(
+		'<portlet:namespace />fileEntryNameInput'
 	);
-	var fileEntryNameInput = window.document.querySelector(
-		'#<portlet:namespace />fileEntryNameInput'
+	const fileEntryRemove = document.getElementById(
+		'<portlet:namespace />fileEntryRemove'
 	);
+	const selectFile = document.getElementById('<portlet:namespace />selectFile');
```
Comment (kresimir-coko) [inline modules/apps/commerce/commerce-product-type-virtual-web/src/main/resources/META-INF/resources/details.jspf:69]: I think we want an empty line between these declarations

**Comment by jonmak08** [comment]
ci:test:sf

**Comment by jonmak08** [comment]
ci:test:relevant

**Review by kresimir-coko — APPROVED** [review]
Apart from the minor inline comment that applies to all files, it looks good! 👍

**Comment by jonmak08** [comment]
ci:test:relevant

**Review by markocikos — APPROVED** [review]
LGTM!

**Comment by markocikos** [comment]
@jonmak08 Please fix conflict and resend

**Comment by jonmak08** [comment]
ci:test:sf

**Comment by jonmak08** [comment]
ci:test:relevant

**Comment by jonmak08** [comment]
ci:test:sf

**Comment by jonmak08** [comment]
ci:reevaluate:1027401_561

**Comment by jonmak08** [comment]
@FabioDiegoMastrorilli please review this PR

**Comment by jonmak08** [comment]
ci:test:relevant

**Comment by FabioDiegoMastrorilli** [comment]
@jonmak08 LGTM :)

**Pull request closed without merging**